dashboard/config/locales/en.yml

```diff
@@ -631,3 +632,18 @@ en:
       tos: "This project has been reported for violating Code.org's <a href='http://code.org/tos'>Terms of Service</a> and cannot be shared with others."
       contact_us: "If you believe this to be an error, please <a href='https://code.org/contact'>contact us.</a>"
       go_to_code_studio: 'Go to Code Studio'
+      report_abuse_form:
+        validation:
+          email: 'Please provide an email address'
+          age: 'Please specify an age'
+          abuse_type: 'Please answer how this content violates the Terms of Service'
+          abuse_detail: 'Please provide details regarding the content you are reporting'
+        abusive_url: 'Abusive URL'
+        abuse_type:
+          question: 'How does this content violate the %{link_start}Terms of Service%{link_end}?'
```

Comment:
@tanyaparker as an FYI. This is a new way of handling html inside of strings that I *think* that I like. It leaves the responsibility of adding the html tags to the caller. I guess we could still get in trouble if the translator did something like leave out link_end, or put it before link_start.